src/include/sof/ipc/common.h

```diff
@@ -68,7 +68,11 @@ struct ipc {
 	struct list_item comp_list;	/* list of component devices */
 
 	/* processing task */
+#if CONFIG_ZEPHYR_TWB_SCHEDULER
```

Comment:
Can we make this a Kconfig for each TWB system task, this way we can transition and back out if needed e.g.
```CONFIG_TWB_IPC_TASK```